### PR TITLE
.rultor.yml: Add merge command

### PR DESCRIPTION
**Pull request by yogupta**
This will help us to use rultor's merge command

Fixes https://github.com/coala/bear-docs/issues/26

### CODE REVIEW

**Comment by sils** [comment]
Hi,

thanks for your contribution!

Please note that this documentation is generated from the [coala-bears](https://github.com/coala/coala-bears/) project. If you are working on rst files, you will very likely want to do changes over there instead!

File: .rultor.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+merge:
+  fast-forward: only
+  script: echo "Nothing to do."
```
Comment (Nosferatul) [inline .rultor.yml]: You don't have newline at the end of the file. Also `Fixes` -> `Closes` in the commit body. Fixes is for bugs

**Comment by Mixih** [comment]
Indent at EOF. Delete everything on line 4

**Comment by yogupta** [comment]
@Nosferatul  asked me to add newline at the end. and you( @Mixih ) are asking me to delete it. I'm confused. what should i do?

**Comment by Mixih** [comment]
no I only asked you to delete all the spaces and whitespace chars on line 4. Leave the newline

**Comment by Mixih** [comment]
Hmm, somehow you got three commits with the same message. Please squash. Also, imo, commit body is unnecessary.

**Comment by jayvdb** [comment]
ack ecfb126

**Comment by jayvdb** [comment]
@rultor merge

**Comment by jayvdb** [comment]
@sils, @yukiisbored , @Mixih , if I understand this error correctly, it is not finding the `.rultor.yml` because it is checking out the repo without applying the pending patch. That feels like a few bugs in rultor, especially that the error message is not clear, bit also it should look for and use the .rultor.yml in the PR.

**Comment by sils** [comment]
yes rultor always takes the base branch one, there's an issue somewhere but you know rultor, not much happening

**Pull request merged**